gallery/identity/ceed-identity.c

```diff
@@ -52,6 +52,6 @@ static int CeedQFunctionInit_Identity(Ceed ceed, const char *requested,
   @brief Register identity QFunction that copies inputs directly into outputs
 **/
 CEED_INTERN int CeedQFunctionRegister_Identity(void) {
-  return CeedQFunctionRegister("Identity", Identity_loc, 1, Identity,
+  return CeedQFunctionRegister("Identity", "gallery/identity/ceed-identity.h:Identity", 1, Identity,
```

Comment:
What path is `Identity_loc` producing? I think we should be able to avoid writing this out manually (and error-prone).

Reply:
Identity_loc currently produces the abspath for the source file. An rpath from libCEED root would be best for this approach and should be doable if we remove some of the abspath stuff from the makefile